### DISCUSSION TITLE
Grid layout with thumbnails for HOME and F1TV, like F1.com Videos layout

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#216] [Feature requests]
Really loving the recently added F1.com Videos integration in MultiViewer with the grid based, thumbnail layout. 

Have you thought about moving the other areas from a list view to a grid view, with large thumbnail images?  Realize this is easier said than done, as there are lots of + - navigation links, incorporated into the existing list view.  I find the thumbnails really very intuitive.

Grid style with thumbnails
![image](https://user-images.githubusercontent.com/22590445/190024265-33ba7dbc-ae48-4822-8580-681a1359a7c8.png)

List style without thumbnails
![image](https://user-images.githubusercontent.com/22590445/190024411-f4960778-6562-4c9d-8db2-73ad043c37c3.png)

**Comment by Jonoz1** [comment]


**Reply by f1multiviewer** [reply]
Please don't bump, just vote:
<img width="99" alt="image" src="https://user-images.githubusercontent.com/106821994/201973744-35101d27-e01d-4e01-b90e-8f7f8217da27.png">
